ocaml/utils/global.mli

```diff
@@ -0,0 +1,56 @@
+[@@@ocaml.warning "+a-9-40-41-42"]
+
```

Comment:
I wonder if we should find some more descriptive name for this module. `Global` is a very overloaded word.

Reply:
My other thought was "instance" or "instance name" but of course that's _also_ overloaded.

Reply:
I like `Global_module`. We might put it in `typing`, yeah: it's very closely tied to `Ident`, which is in `typing`. It does end up in .cmx files for use by `-instantiate`, but in a sense that's still the front end (i.e., it's used to generate Lambda code).

Reply:
Renamed it to `Global_module` and put it in `typing`.